### PR TITLE
bus-unification

### PR DESCRIPTION
**Pull request by SirDomin**

### CODE REVIEW

File: src/Resources/config/services.xml [hunk 1]
```diff
@@ -28,11 +28,11 @@
 
         <service id="Sylius\RefundPlugin\Refunder\OrderItemUnitsRefunder">
             <argument type="service" id="Sylius\RefundPlugin\Creator\RefundCreator" />
-            <argument type="service" id="sylius_refund_plugin.event_bus" />
+            <argument type="service" id="sylius_default.bus" />
         </service>
         <service id="Sylius\RefundPlugin\Refunder\OrderShipmentsRefunder">
             <argument type="service" id="Sylius\RefundPlugin\Creator\RefundCreator" />
-            <argument type="service" id="sylius_refund_plugin.event_bus" />
+            <argument type="service" id="sylius_default.bus" />
```
Comment (lchrusciel) [inline src/Resources/config/services.xml:35]:
```suggestion
            <argument type="service" id="sylius.event_bus" />
```

File: src/Resources/config/app/config.yml [hunk 2]
```diff
@@ -26,13 +26,6 @@ sylius_mailer:
             subject: Units refunded
             template: "@SyliusRefundPlugin/Email/unitsRefunded.html.twig"
 
-framework:
-    messenger:
-        buses:
-            sylius_refund_plugin.command_bus: ~
-            sylius_refund_plugin.event_bus:
-                default_middleware: allow_no_handlers
```
Comment (lchrusciel) [inline src/Resources/config/app/config.yml:34]: We may be forced to keep following config:
```yaml
framework:
    messenger:
        buses:
            sylius_event.bus:
                middleware:
                    - 'allow_no_handler'
```

**Review by AdamKasp — APPROVED** [review]

**Review by lchrusciel — APPROVED** [review]

File: src/Resources/config/services/event_bus.xml [hunk 3]
```diff
@@ -5,32 +5,32 @@
...
 
         <service id="Sylius\RefundPlugin\ProcessManager\RefundPaymentProcessManager">
             <argument type="service" id="Sylius\RefundPlugin\StateResolver\OrderFullyRefundedStateResolver" />
             <argument type="service" id="Sylius\RefundPlugin\Provider\RelatedPaymentIdProviderInterface" />
             <argument type="service" id="Sylius\RefundPlugin\Factory\RefundPaymentFactory" />
             <argument type="service" id="doctrine.orm.default_entity_manager" />
-            <argument type="service" id="sylius_refund_plugin.event_bus" />
-            <tag name="messenger.message_handler" />
+            <argument type="service" id="sylius_event.bus" />
+            <tag name="messenger.message_handler" bus="sylius_default.bus" />
```
Comment (GSadee) [inline src/Resources/config/services/event_bus.xml:33]:
```suggestion
            <tag name="messenger.message_handler" bus="sylius_event.bus" />
```
?

**Review by pamil — APPROVED** [review]

**Review by GSadee — COMMENTED** [review]
Changes of buses should be mentioned in UPGRADE file

File: src/Resources/config/services/event_bus.xml [hunk 4]
```diff
@@ -5,32 +5,32 @@
         <defaults autowire="false" autoconfigure="false" public="true" />
 
         <service id="Sylius\RefundPlugin\Listener\CreditMemoGeneratedEventListener">
-            <argument type="service" id="sylius_refund_plugin.event_bus" />
-            <tag name="messenger.message_handler" />
+            <argument type="service" id="sylius.event_bus" />
```
Comment (GSadee) [inline src/Resources/config/services/event_bus.xml:8]:
```suggestion
            <argument type="service" id="sylius.command_bus" />
```

File: src/Resources/config/services/event_bus.xml [hunk 5]
```diff
@@ -5,32 +5,32 @@
...
         <service id="Sylius\RefundPlugin\Listener\ShipmentRefundedEventListener">
             <argument type="service" id="Sylius\RefundPlugin\StateResolver\OrderPartiallyRefundedStateResolver" />
-            <tag name="messenger.message_handler" />
+            <tag name="messenger.message_handler" bus="sylius.event_bus" />
         </service>
 
         <service id="Sylius\RefundPlugin\ProcessManager\CreditMemoProcessManager">
-            <argument type="service" id="sylius_refund_plugin.command_bus" />
-            <tag name="messenger.message_handler" />
+            <argument type="service" id="sylius.event_bus" />
```
Comment (GSadee) [inline src/Resources/config/services/event_bus.xml:23]:
```suggestion
            <argument type="service" id="sylius.command_bus" />
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, @SirDomin! :tada:

**Pull request merged**